Content.Server/GameObjects/Components/Nutrition/FoodContainerComponent.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Shared.GameObjects.Components.Nutrition;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Nutrition
+{
+    /// <summary>
+    /// This container acts as a master object for things like Pizza, which holds slices.
+    /// </summary>
+    /// TODO: Perhaps implement putting food back (pizza boxes) but that really isn't mandatory.
+    /// This doesn't even need to have an actual Container for right now.
+    [RegisterComponent]
+    public sealed class FoodContainer : SharedFoodContainerComponent, IUse
+    {
+#pragma warning disable 649
+        [Dependency] private readonly IRobustRandom _random;
+        [Dependency] private readonly IEntityManager _entityManager;
+#pragma warning restore 649
+        public override string Name => "FoodContainer";
+
+        private AppearanceComponent _appearance;
+        private Dictionary<string, int> _prototypes;
+        private uint _capacity;
```

Comment:
I don't see the point of storing this as a uint.

Reply:
It will never be negative.

Reply:
those are some famous last words right there

Reply:
Yeah but you're returning it as an int in public API so:
1. If you actually need that extra space, you have a bug
2. If you don't need it (you don't), you shouldn't be using a uint because it's annoying.